pkg/e2e/e2e.go

```diff
@@ -158,6 +159,17 @@ func beforeSuite() bool {
 		log.Println("Skipping health checks as requested")
 	}
 
+	//Creates a configmap for PrCheck to use as a psudo queue
```

Comment:
I think having a config is useful for testing (and other non-prow-invocations) but before doing the check on line 163 we could also look at the `JOB_NAME` env var and see if it matches `pull-ci-openshift-osde2e-main-verify` -- Prow will always pass the name of the job into the pod it's running. :) 

Then once this is merged we don't need to go and touch another repo for this to take effect.

Reply:
Closing this change request since we changed the approach to be ocm.Property dependent.